src/peerslist.test.js

```diff
@@ -0,0 +1,44 @@
+const {getSwarms} = require('./peerslist');
```

Comment:
Please add unit tests covering the following:

- Verify attempts to connect to appropriate nodes given data in the format that the CPR will return (should be able to mock the actual connection attempt)
- Verify desirable behaviour in case of connection failure to the CPR

Reply:
> Verify attempts to connect to appropriate nodes given data in the format that the CPR will return (should be able to mock the actual connection attempt)

I need clarification on what we are intending to verify, as the behavior described does not align to any individual software module or layer besides the WebSocket constructor. It sounds like we want an integration test with a mocked swarm.

Reply:
If that's the case, I'll vouch for mocking the ports themselves as those are the most natural interfaces.

Reply:
I'm not familiar with the internal architecture and mock-able points, but the intent is to assert something like "Given a valid document from CPR, the client will attempt to make connections to the list of nodes given by the CPR"